_layouts/default.html

```diff
@@ -12,6 +12,7 @@
         {%- include sidebar.html -%}
       </aside>
       <div class="page-content">
+        <a href="{{ site.github.repository_url }}/tree/master/{{ page.path }}">{{ site.data.common[page.lang].button.edit }}</a>
```

Comment:
IMHO it would be better to have the link right-aligned. What do you think?
<img width="881" alt="Capture d’écran 2019-10-10 à 12 54 33" src="https://user-images.githubusercontent.com/1162230/66563040-34183b80-eb5d-11e9-8a1e-dcfcb65962c9.png">

Reply:
I agree, the top of the top would be - in my opinion - having a breadcrumb on the left and the button on the same line on the right. 
I can align the button on the left right now and we can discuss about other design improvements later if you want :) I can also give the link a more "button" aspect if you want.

Reply:
The breadcrumb is another pull request 🙂 

Yes, a button aspect would be cool 👍 
There are no requirements on design, do your best, just try to stay consistent with the rest, and it will be totally fine. FYI, we are using the [Minima theme for Jekyll](https://jekyll.github.io/minima/), maybe you can find some predefined styles there?

Reply:
Ok great, I will check that and make other PR :) 
For now, I will add the missing styles on the button.